test/vm.install

```diff
@@ -15,3 +15,4 @@ fi
 # grab a few images to play with; tests run offline, so they cannot download images
 podman pull busybox
 podman pull docker.io/alpine
+# podman rm eager_murdock
```

Comment:
Please don't commit such stuff.

Reply:
Right, you can just squash that into the existing commit on the next force-push. This  PR needs some more involved rework anyway (see above).